tservice/sys/build.rs

```diff
@@ -2,39 +2,58 @@
 //! Builds the FFI function bindings for tservice, (trusted service) of the
 //! Intel Intel SGX SDK
 
-use bindgen::Builder;
 use cargo_emit::{rustc_link_lib, rustc_link_search};
 
+const SERVICE_FUNCTIONS: &[&str] = &[
+    "sgx_create_report",
+    "sgx_self_report",
+    "sgx_get_key",
+    "sgx_aligned_free",
+    "sgx_aligned_malloc",
+    "sgx_get_aligned_ptr",
+    "sgx_verify_report",
+    "sgx_verify_report2",
+    "sgx_derive_target_from_report",
+    "sgx_dh_init_session",
+    "sgx_dh_responder_gen_msg1",
+    "sgx_dh_responder_proc_msg2",
+    "sgx_LAv1_initiator_proc_msg1",
+    "sgx_LAv1_initiator_proc_msg3",
+    "sgx_LAv2_initiator_proc_msg1",
+    "sgx_LAv2_initiator_proc_msg3",
+    "sgx_self_target",
+    "sgx_seal_data",
+    "sgx_seal_data_ex",
+    "sgx_unseal_data",
+    "sgx_mac_aadata",
+    "sgx_mac_aadata_ex",
+    "sgx_unmac_aadata",
+    "sgx_seal_data_iv",
+    "sgx_unseal_data_helper",
+    "sgx_calc_sealed_data_size",
+    "sgx_get_add_mac_txt_len",
+    "sgx_get_encrypt_txt_len",
+];
```

Comment:
`sgx_seal_data_iv` and `sgx_unseal_data_helper` appear to be internal functions (declared in `sdk/tseal/tSeal_internal.h`)

Reply:
`sgx_derive_target_from_report` appears to be an internal function (not declared in any header)

Reply:
removed private/internal functions from the bindings list